CONTRIBUTING.md

```diff
@@ -0,0 +1,107 @@
+
+```console
+    $ git@github.com:akshatnema/jobie.git (ssh key) or $ https://github.com/akshatnema/jobie.git (https)
+    $ cd jobie  
+```
+<h3>Step 2:Branch</h3>
+To keep your development environment organized, create local branches to hold your work. These should be branched directly off of the main branch.
+
+```console
+   $ git checkout -b my-branch -t upstream/main
+   $ git remote add upstream https://github.com/akshatnema/jobie.git
+   $ git fetch upstream
+```
+<h3>Step 3:Build</h3>
+<p>Now we are ready to build the project , but first of all we need to install NodeJS in our local system.<p>
+    
+```console
+    $ sudo apt install nodejs
+```
+ (for Debian Based system only)
+ <p>Inside the JobiE folder run the following commands to install all the dependencies.<p>
+     
+```console
+    $ cd server
+    $ npm install
+``` 
+(To install all the server side dependencies)
+<p>Similarly for the front-end side, use the following commands in the JobiE folder<p>
+     
+```console
+    $ cd web
+    $ npm install
+```
+<h3>Step 4:Setting up development variables</h3>
+Now we need to create a .env file which will contain our development variables such as Atlas string(URI) and Twitter Bearer Token for the twitter API.
+
+```console
+    $ cd server
+    $ touch .env
+    $ nano .env
+```
+<p>Note:You can use any text editor in place of nano.</p>
+
+<p>Now inside the .env file use your development variables as shown below.</p>
+
+```console
+    ATLAS_URI="mongodb+srv://user_name:password@cluster0.bscef8f.mongodb.net/?retryWrites=true&w=majority"
+    TWITTER_API="AAAAAAAAAAAAAAAAAAAAANAUSgEAAAAA%2BwvPQqv%xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxdCH93sdrzv"
+```
+
+<p><b>Twitter Bearer Token</b>: Click <a href="https://developer.twitter.com/en/docs/developer-portal">here.</a></p>
+<p><b>ATLAS URI</b>: Click <a href="https://account.mongodb.com/account/login?n=%2Fv2%2F62f7e246576e344fad111a1c&nextHash=%23clusters">here</a>.</p>
+
+<h3>Step 5:Start Developing</h3>
+<p>For all the users working on the frontend. You can make changes and load JobiE locally by running the following inside the web directory.</p>
+
+```console
+   $ npm start
+```
+<p>Similarly for the server side, use the same command inside the sever directory.</p>
+Your site will be hosted locally at: <b>http://localhost:8000/</b>
+
+<h3>Step 6:Commit</h3>
+<p>It is recommended to keep your changes grouped logically within individual commits. Commits are a great way to help your fellow contributors keep check of the changes you made to the project.</p>
+
+```console
+   $ git add my/changed/file
+   $ git commit -m "changes-made"
+```
+<h3>Step 7:Rebase</h3>
+<p>Once you have committed your changes, it is a good idea to use git rebase (not git merge) to synchronize your work with the main repository.</p>
+
+```console
+   $ git fetch upstream
+   $ git rebase upstream/main
+```
+<p>This ensures that your working branch has the latest changes from main.</p>
+<h2>Testing</h2>
```

Comment:
Presently, there are no tests available to check the linting or bugs in the changes. So, better you right now remove it and we will add this afterwards when tests will be added.